src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,7 +5,43 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int bodyWidth;
+    private int height;
+    private int weight;
```

Comment:
Consider adding an access modifier to the 'weight' field. Without an access modifier, the field has package-private access, which is less restrictive than private access.